ISPN-13861 Support bare protobuf annotations (#10082)
infinispan/infinispan: documentation/src/main/asciidoc/topics/ref_protostream_annotations.adoc

{brandname} does not recommend using this annotation because it is no longer supported by Google and is likely to be removed in future.
====

[discrete]

> == Other annotations
> Any other annotations which are present on classes, fields and methods, will be copied by Protostream as comments in the generated schema, without the need to wrap them within `@ProtoDoc` annotations.
> 
> Use this capability to add indexing annotations to your entities.

Reviewer: ```suggestion
== Other annotations
{brandname} copies any other annotations on classes, fields, and methods as comments in the generated Protobuf schema.
This includes indexing annotations such as `@Indexed` and `@Field`.
```
